full/Dockerfile

```diff
@@ -208,14 +208,14 @@ RUN curl -fsSL https://github.com/pyenv/pyenv-installer/raw/master/bin/pyenv-ins
         echo 'eval "$(pyenv virtualenv-init -)"'; } >> /home/gitpod/.bashrc.d/60-python \
     && pyenv update \
     && pyenv install 2.7.17 \
-    && pyenv install 3.7.7 \
-    && pyenv global 2.7.17 3.7.7 \
+    && pyenv install 3.8.2 \
+    && pyenv global 2.7.17 3.8.2 \
     && python2 -m pip install --upgrade pip \
     && python3 -m pip install --upgrade pip \
     && python3 -m pip install --upgrade \
         setuptools wheel virtualenv pipenv pylint rope flake8 \
         mypy autopep8 pep8 pylama pydocstyle bandit notebook \
-        python-language-server[all]==0.25.0 twine \
```

Comment:
Why are you dropping this

Reply:
@RDIL I don't think it's used anywhere, so no need to pre-install it.

Do you use it somewhere?

Reply:
@RDIL, we're not using the `@theia/python` extension anymore, which was the reason to add it here in the first place.

Reply:
Okay, understandable.